nuage-tripleo-heat-templates/docker/services/nova-compute-avrs.yaml

```diff
@@ -0,0 +1,304 @@
+
+  MySQLClient:
+    type: /usr/share/openstack-tripleo-heat-templates/puppet/services/database/mysql-client.yaml
+
+  NovaComputeCommon:
+    type: /usr/share/openstack-tripleo-heat-templates/docker/services/nova-compute-common.yaml
+    properties:
+      EndpointMap: {get_param: EndpointMap}
+      ServiceData: {get_param: ServiceData}
+      ServiceNetMap: {get_param: ServiceNetMap}
+      DefaultPasswords: {get_param: DefaultPasswords}
+      RoleName: {get_param: RoleName}
+      RoleParameters: {get_param: RoleParameters}
+
+  NovaComputeAvrsBase:
+    type: /usr/share/openstack-tripleo-heat-templates/puppet/services/nova-compute.yaml
+    properties:
+      EndpointMap: {get_param: EndpointMap}
+      ServiceData: {get_param: ServiceData}
+      ServiceNetMap: {get_param: ServiceNetMap}
+      DefaultPasswords: {get_param: DefaultPasswords}
+      RoleName: {get_param: RoleName}
+      RoleParameters: {get_param: RoleParameters}
+
+  NovaComputeAvrsDockerBase:
+    type: /usr/share/openstack-tripleo-heat-templates/docker/services/nova-compute.yaml
+    properties:
+      EnableInstanceHA: {get_param: EnableInstanceHA}
+      DockerNovaComputeImage: {get_param: DockerNovaComputeAvrsImage}
+      EndpointMap: {get_param: EndpointMap}
+      ServiceData: {get_param: ServiceData}
+      ServiceNetMap: {get_param: ServiceNetMap}
+      DefaultPasswords: {get_param: DefaultPasswords}
+      RoleName: {get_param: RoleName}
+      RoleParameters: {get_param: RoleParameters}
+
+  NovaLogging:
+    type: OS::TripleO::Services::Logging::NovaCommon
+    properties:
+      DockerNovaImage: {get_param: DockerNovaComputeAvrsImage}
+      NovaServiceName: 'compute'
+
+conditions:
+  enable_instance_ha: {equals: [{get_param: EnableInstanceHA}, true]}
+
+outputs:
+  role_data:
+    description: Role data for the Nova Compute service.
+    value:
+      service_name: {get_attr: [NovaComputeAvrsBase, role_data, service_name]}
+      cellv2_discovery: true
+      config_settings:
+        map_merge:
+          - get_attr: [NovaComputeAvrsBase, role_data, config_settings]
+          - get_attr: [NovaLogging, config_settings]
+      logging_source: {get_attr: [NovaComputeAvrsBase, role_data, logging_source]}
+      logging_groups: {get_attr: [NovaComputeAvrsBase, role_data, logging_groups]}
+      service_config_settings: {get_attr: [NovaComputeAvrsBase, role_data, service_config_settings]}
+      puppet_config:
+        config_volume: nova_libvirt
+        puppet_tags: nova_config,nova_paste_api_ini
+        step_config:
+          list_join:
+            - "\n"
+            - - {get_attr: [NovaComputeAvrsBase, role_data, step_config]}
+              - {get_attr: [MySQLClient, role_data, step_config]}
+        config_image: {get_param: DockerNovaLibvirtAvrsConfigImage}
+      kolla_config:
+        get_attr: [NovaComputeAvrsDockerBase, role_data, kolla_config]
+      docker_config_scripts:Making Re
```

Comment:
There appears to be some extra items here "Making Re" probably needs to be removed

Reply:
Thanks. Updated the PR.